src/layout-persistence/dotnet/src/MorganStanley.ComposeUI.LayoutPersistence/LayoutManager.cs

```diff
@@ -0,0 +1,41 @@
+﻿/*
+ * Morgan Stanley makes this available to you under the Apache License,
+ * Version 2.0 (the "License"). You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership. Unless required by applicable law or agreed
+ * to in writing, software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+
+using MorganStanley.ComposeUI.LayoutPersistence.Abstractions;
+
+namespace MorganStanley.ComposeUI.LayoutPersistence;
+
+public class LayoutManager
```

Comment:
I'm not convinced this class is necessary.
Wouldn't it make more sense to make the ILayoutPersistence methods generic and take an ILayoutSerializer as a dependency of FileLayoutPersistence?